src/index.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>ToDo App</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS v5.2.1 -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+</head>
+
+<body>
+    <header>
+        <!-- place navbar here -->
+    </header>
+    <main>
+        <div class="container">
+            <div class="row">
+                <div id="myCard">
+                    <div class="row">
+                        <h1>ToDo List</h1>
+                        <form id="add-task-form">
+                            <div class="input-group">
+                                <input type="text" id="task-input" class="no-outline" placeholder="new task ..."
+                                    autofocus>
+                                <button type="submit" hidden>Add Task</button>
+                            </div>
+                            <div>
+                                <ul id="todo-list">
+                                </ul>
+                            </div>
+                        </form>
+                    </div>
+                    <div class="row" style="background-color: lightgray;"> <button id="clear-completed-button" class="btn" style="color: black;">Clear all
```

Comment:
Kindly avoid using inline styles and keep your style definition in a separate CSS file.